aio_pika/tools.py

```diff
@@ -2,6 +2,38 @@
 from functools import partial
 
 
+def iscoroutinepartial(fn):
```

Comment:
As well there is no sense to make regular function `asyncio.coroutine` in runtime